filesystem.go

```diff
@@ -570,6 +599,32 @@ func (fs *filesystem) storeContainerProcess(podID, containerID string, process P
 	return fs.storeContainerResource(podID, containerID, processFileType, process)
 }
 
+func (fs *filesystem) fetchContainerMounts(podID, containerID string) ([]Mount, error) {
+	if podID == "" {
+		return []Mount{}, errNeedPodID
+	}
+
+	if containerID == "" {
+		return []Mount{}, errNeedContainerID
+	}
+
+	data, err := fs.fetchResource(false, podID, containerID, mountsFileType)
+	if err != nil {
+		return []Mount{}, err
+	}
+
+	switch mounts := data.(type) {
+	case []Mount:
+		return mounts, nil
+	default:
+		return []Mount{}, fmt.Errorf("Unknown mounts type : [%T]", mounts)
```

Comment:
I am sorry I have deleted your comment by mistake. But I agree that you need this way to display the type.